### PR TITLE
Use log functions of core framework on test/e2e/freamwork/psp.go

### PR DESCRIPTION
**Pull request by clarklee92**
Signed-off-by: clarklee92 <clarklee1992@hotmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
I move e2e/framework/psp/psp.go to  e2e/framework/psp.go. This makes psp.go to use log functions and avoiding circular dependencies if `PodSecurityPolicy` related code may be not in a new sub-package.😊

**Which issue(s) this PR fixes**:

Ref: #81427

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/framework/framework.go [hunk 1]
```diff
@@ -401,7 +400,7 @@ func (f *Framework) CreateNamespace(baseName string, labels map[string]string) (
 	f.AddNamespacesToDelete(ns)
 
 	if err == nil && !f.SkipPrivilegedPSPBinding {
-		e2epsp.CreatePrivilegedPSPBinding(f.ClientSet, ns.Name)
+		CreatePrivilegedPSPBinding(f.ClientSet, ns.Name)
```
Comment (oomichi) [inline test/e2e/framework/framework.go]: This part is a really hard point to remove dependency to e2epsp subpackage from the core framework.
I tried to remove this part, but I found it is so difficult and gave up.

**Review by oomichi — COMMENTED** [review]
TBH, this is difficult to make a decision because the subpackage of psp is not so bad.
But from experience I tried to remove invalid dependency, I tend to take this.
Anyways I'd like to see the result of CI tests.

/ok-to-test

File: test/e2e/framework/framework.go (hunk 1 above)
Comment (clarklee92) [reply test/e2e/framework/framework.go]: The CI tests passed. I think move psp.go under framework is resonable and harmless. The intention is that the framework does not use subpackages and all subpackages use the same logging framework. There are only two functions called by `framework` and `auth`, the function call from `framework` is hard to remove as you say , why don't we make psp.go as a part of framework.

**Comment by oomichi** [comment]
Thanks for doing this.

/lgtm
/approve

**Pull request merged**